mapserver/maps/views.py

```diff
@@ -57,7 +61,7 @@ def search_form(request):
                         lat=str(result.lat),
                         long=str(result.long),
                         accel=str(result.z_axis),
-                        generator=result.generator.pk,
+                        generator="",
```

Comment:
This might leave a field with these quotes in the info boxes. More specifically, the titles of the info boxes are the generator info I think. Change it to "Marker"?

Otherwise, doesnt crash anything. Not tested yet, will get back to you soon.